library/build.gradle

```diff
@@ -2,9 +2,10 @@ apply plugin: "com.android.library"
 
 android {
     compileSdkVersion 23
-    buildToolsVersion "23.0.2"
+    buildToolsVersion "25.0.3"
```

Comment:
I'd prefer to keep the version numbers in sync with `https://github.com/AntennaPod/AntennaPod/blob/develop/build.gradle`